scss/nes.scss

```diff
@@ -1,8 +1,10 @@
 @charset "utf-8";
 
-/*!
-* NES.css v0.0.2(alpha)
-*/
+/*****************************************************************************\
+  NES.css Framework
+  Version: development
+#{build-data()}
+\*****************************************************************************/
```

Comment:
I'd like to leave credit title(comment) when minified. 🙏 

so could you change `/******\`-`\*****/` to `/*!`-`*/`. 🙇 
https://github.com/jakubpawlowicz/clean-css#how-to-preserve-a-comment-block

Reply:
Ah, of course! I'll push an update for that shortly. 😉